### ISSUE TITLE
buttons should be readable by screen readers

### ISSUE THREAD

**Issue by techieshark** [CodeforAustralia/school-finder#203]
@reekypete [pointed out](https://github.com/CodeforAustralia/school-finder/pull/194/commits/16bd14ddb233001a5426c12e411588946c4c1a8a#diff-eacf331f0ffc35d4b482f1d15a887d3bR60) that button controls could have labels which are more self contained (so screen reader users wouldn't have to have surrounding context to make use of the buttons). 

A couple thoughts:
- now that we're using more semantically correct `<button>`s rather than `<a>`s, we'll need something other than `title` attribute to make this work.
- I came across documentation about what to do in this case while doing research; I'll have to dig that up. We might use `<span class="sr-only">` (screen reader only) to add that context into the button text, or use ARIA labelling. 
- this issue may relate to all buttons we use, but we should at least start with the primary / secondary school buttons at the top.
